templates/default/login.html

```diff
@@ -67,6 +67,14 @@
 		</span>
 		<form method="post" name="loginform" action="{$target}">
 			<table>
+                <tr>
+                    <td class="text-center">
+                        {icon name="description"}
+                    </td>
+                    <td class="bold">
+                        {$instance_label}
+                    </td>
+                </tr>
```

Comment:
Czyli instance_label zawsze będzie prezentowany?
Moim zdaniem to nie powinno mieć wartości domyślnej i wtedy w ogóle nie być prezentowane.

Reply:
Tak - domyślnie hostname pokazywane jest przy logowaniu. Zauważyłem, że w trybie pełnego ekranu nie widać gdzie się logujemy jeśli mamy kilka instancji. Podobnie na telefonach po schowaniu paska adresowego.

Reply:
Poprawione wg zaleceń.